homeassistant/components/pi_hole/__init__.py

```diff
@@ -129,9 +129,26 @@ async def async_update_data() -> None:
                         raise ConfigEntryAuthFailed
         except HoleError as err:
             if str(err) == "Authentication failed: Invalid password":
+                _LOGGER.error(
+                    "Pi-hole %s at host %s, reported an invalid password",
+                    name,
+                    host,
+                )
                 raise ConfigEntryAuthFailed from err
+            _LOGGER.error(
+                "Pi-hole %s at host %s, update failed with HoleError: %s",
+                name,
+                host,
+                err,
+            )
             raise UpdateFailed(f"Failed to communicate with API: {err}") from err
```

Comment:
For these cases, shouldn't it already log what is happening because of the `from err`?